client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
IMO it's less maintenance to change the value in a single place by maintaining convention
than updating a value in 2 places, since you might not even know where is the second place